packages/kv/README.md

```diff
@@ -69,3 +69,49 @@ await kv.set('key', 'value');
 ## Docs
 
 See the [documentation](https://www.vercel.com/docs/storage/vercel-kv) for details.
+
+## A note for Vite users
+
+`@vercel/kv` reads database credentials from the environment variables on `process.env`. In general, `process.env` is automatically populated from your `.env` file during development, which is created when you run `vc env pull`. However, Vite does not expose the `.env` variables on `process.env.`
+
+You can fix this in **one** of following two ways:
+
+1. You can populate `process.env` yourself using something like `dotenv-expand`:
+
+```shell
+pnpm install --save-dev dotenv dotenv-expand
+```
+
+```js
+// vite.config.js
+import dotenvExpand from 'dotenv-expand';
+import { loadEnv, defineConfig } from 'vite';
+
+export default defineConfig(({ mode }) => {
+  // This check is important!
+  if (mode === 'development') {
+    const env = loadEnv(mode);
```

Comment:
when testing locally I had to change it to 
```
const env = loadEnv(mode, process.cwd())
```

Reply:
This didn't work for me. What did work:

```
    if (mode === 'development') {
        const env = loadEnv(mode, process.cwd(), '');
        dotenvExpand.expand({parsed: env});
    }
```

Reply:
Hang it, I copied the wrong example. One sec...

Reply:
Thanks @IsaacBreen, didn't see your comment until I had already fixed the README! He's right @correttojs, I forgot you have to specify the third parameter (it's the prefix to load, defaults to `VITE_`).

Reply:
You're welcome! The `{parsed: env}` part is important too. Vite's `loadEnv` [returns an object of env vars](https://github.com/vitejs/vite/blob/d95a9af5c1aa9babecd710ef2d341a7839b41daf/packages/vite/src/node/env.ts#L54), but `dotenvExpand.expand(config)` [looks for them under `config.parsed`](https://github.com/motdotla/dotenv-expand/blob/4ac51eac991d6ed87d1d56f35f4b85c77bb4ed5f/lib/main.js#L43) .

Reply:
lol, thanks -- it's obviously too late over here for me to be useful <picture data-single-emoji=":upside-down-cowboy:" title=":upside-down-cowboy:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/upside-down-cowboy/47996501201dadf8.png" alt=":upside-down-cowboy:" align="absmiddle"></picture>